app/controllers/state_file/questions/w2_controller.rb

```diff
@@ -15,6 +15,11 @@ def prev_path
       end
 
       def edit
+        if StateFile::StateInformationService.check_box_16(current_state_code)
+          current_intake.confirmed_w2_ids.append(@w2.id)
```

Comment:
[pebble] if this page gets visited multiple times, will you add the id multiple times into the column as well? I think we should not add if it already exists